python_modules/dagster/dagster/_cli/workspace/cli_target.py

```diff
@@ -334,14 +334,18 @@ def python_module_option(allow_multiple):
     )
 
 
+def working_directory_option():
+    return click.option(
+        "--working-directory",
+        "-d",
+        help="Specify working directory to use when loading the repository or job",
+        envvar="DAGSTER_WORKING_DIRECTORY",
+    )
```

Comment:
side-quest: i think there might be click arguments that validate the value is a directory